lib/union/Customer.js

```diff
@@ -0,0 +1,53 @@
+const request = require('./common/request');
+
+exports.CustomerEnquiry = async(data) => {
+    try {
+        return await request.trigger({
+            method: 'POST',
+            path: '/union/secured/cbacustomerenquiry',
+            credentials: { ...data },
+            payload: data.payload,
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.CustomerAndAccountEnquiry = async(data) => {
```

Comment:
Should be `exports.CustomerAccountEnquiry`

Reply:
Changed.